components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,846 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import React, { useEffect, useState } from "react";
+import { countries } from 'countries-list';
+import ContextMenu, { ContextMenuEntry } from "../components/ContextMenu";
+import { SettingsPage } from "./SettingsPage";
+import { getGitpodService } from "../service/service";
+import ThreeDots from '../icons/ThreeDots.svg';
+import Modal from "../components/Modal";
+import { AssigneeIdentifier, TeamSubscription, TeamSubscriptionSlotResolved } from "@gitpod/gitpod-protocol/lib/team-subscription-protocol";
+import { Currency, Plan, Plans } from "@gitpod/gitpod-protocol/lib/plans";
+import { ChargebeeClient } from "../chargebee/chargebee-client";
+import { copy, exclamation } from "../images";
```

Comment:
Heads up: Images should be imported directly (we're removing the eager-loading `images` module in https://github.com/gitpod-io/gitpod/commit/30f72c888a4da097271c923f1d35336aa8169f8c)

```suggestion
import copy from '../images/copy.svg';
import exclamation from '../images/exclamation.svg';
```

Reply:
somehow telepresence seems to be broken since :-(